lib/possible_bipartition.rb

```diff
@@ -1,4 +1,31 @@
 
+###Used leetcode as a reference because my brain is a fried egg, 
+###but I do understand why this solution works. 
+
 def possible_bipartition(dislikes)
```

Comment:
You're basically using depth-first-search, and coloring nodes 1 then -1, and if you have a node colored 1 next to another node colored 1, you can't bipartition the graph.  Similarly if you have two nodes colored -1, you can't do a bipartition.